### ISSUE TITLE
PHP-DI 6.x causes bug due to removed function

### ISSUE THREAD

**Issue by chrisvanpatten** [gettyimages/gettyimages-api_php#17]
PR #16 enabled usages of newer versions of PHP-DI, however it also introduced a bug due to the removal of the `DI\object()` function in PHP-DI 6.x:

```
[30-Sep-2021 14:10:29 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function DI\Object() in /var/www/html/vendor/gettyimages/gettyimages-api/src/GettyImages_Client.php:102
Stack trace:
#0 /var/www/html/vendor/gettyimages/gettyimages-api/src/GettyImages_Client.php(121): GettyImages\Api\GettyImages_Client->__construct()
#1 /var/www/html/app/getty/container.php(41): 
```

(partial stack trace)

Note the removal in PHP-DI's upgrade guide: https://github.com/PHP-DI/PHP-DI/blob/master/doc/migration/6.0.md#diobject

**Comment by ChrisSimmons** [comment]
The changes from #16 were reverted and a new release (v4.1.2) was created.

**Comment by ChrisSimmons** [comment]
Closing this but please re-open if you see errors on your side, @chrisvanpatten

**Comment by ChrisSimmons** [comment]
php-di upgraded to v6 with the fix as suggested in the linked document.  Released v4.1.3 of this package.
